static/field.json

```diff
@@ -13,5 +13,8 @@
     ],
     interactableData: [
         {x: 20, y: 20, w: 10, h: 10}
+    ],
+    rampsData: [
+        {x: 0, y: 50, w: 30, h: 40, highSide: "up", incline: 15, color: "black"}
```

Comment:
Let's maybe not add this to the default field for now, and add it in later with the 2021 game field